scheduler/src/cook/kubernetes/api.clj

```diff
@@ -800,7 +801,10 @@
         params-env (build-params-env parameters)
         progress-env (task/build-executor-environment job)
         checkpoint-env (checkpoint->env checkpoint)
-        main-env-base (merge environment params-env progress-env sandbox-env checkpoint-env)
+        metadata-env {"COOK_COMPUTE_CLUSTER_NAME" compute-cluster-name
+                      "COOK_POOL" (pool/pool-name-or-default pool-name)
+                      "COOK_SCHEDULER_REST_URL" (config/scheduler-rest-url)}
```

Comment:
COOK_SCHEDULER_REST_URL is already used by the sidecar. can't rename to COOK_SCHEDULER_URL right now